Makefile

```diff
@@ -15,4 +15,4 @@ build:
 push:
 	docker push $(REGISTRY)/alicloud-controller-manager:$(VERSION)
 
-.PHONY: clean binary build push
```

Comment:
Thanks for the patch! Cloud you keep this **push** in .PHONY please, it won't change the default behavior of `make`.